packages/primitives/test/emotion-primitives.test.js

```diff
@@ -140,17 +140,17 @@ describe('Emotion primitives', () => {
     expect(tree).toMatchSnapshot()
   })
 
-  it('innerRef', () => {
-    const Text = styled('p')`
-      color: hotpink;
-    `
-    let ref = React.createRef()
-    const rootNode = document.createElement('div')
-
-    render(<Text innerRef={ref} id="something" />, rootNode)
-    expect(ref.current).toBe(rootNode.querySelector('#something'))
-    unmountComponentAtNode(rootNode)
-  })
+  // it('innerRef', () => {
+  //   const Text = styled('p')`
+  //     color: hotpink;
+  //   `
+  //   let ref = React.createRef()
+  //   const rootNode = document.createElement('div')
+
+  //   render(<Text innerRef={ref} id="something" />, rootNode)
+  //   expect(ref.current).toBe(rootNode.querySelector('#something'))
+  //   unmountComponentAtNode(rootNode)
+  // })
```

Comment:
Any reason this was commented out?

Reply:
Hm.. can't recall may be its related to emotion primitives PR work. Anyway, I'll uncomment and update the test.